pyproject.toml

```diff
@@ -64,8 +64,8 @@ dependencies = [
     "kornia>=0.6.9",
     # timm models for image classification and regression
     "timm>=0.9.2",
-    # torchseg for segmentation, pixelwise regression
-    "torchseg>=0.0.1a1",
+    # smp for segmentation, pixelwise regression
+    "segmentation-models-pytorch>=0.4.0",
```

Comment:
Your minimum versions are likely too strict. Even SMP now supports older timm. Older SMP likely works for you too.